pkg/shim/runtimeoptions/runtimeoptions.go

```diff
@@ -15,3 +15,9 @@
 
 // Package runtimeoptions contains the runtimeoptions proto.
 package runtimeoptions
+
+import proto "github.com/gogo/protobuf/proto"
+
+func init() {
+	proto.RegisterType((*Options)(nil), "runtimeoptions.v1.Options")
```

Comment:
I think a comment is necessary that says that this is a temporary fix to allow the shim to work with containerd 1.5+. 

also create a new Github issue to upgrade containerd dependency to 1.5 and add a TODO comment pointing to it here.

Something like:
```
// This is a temporary fix to allow us to be compatible with containerd 1.5+. It works
// because while the package has changed the Options proto itself hasn't. It can be
// removed once we update runtimeoptions.proto to the version containerd 1.5 and
// add the corresponding generated code.
// TODO(gvisor.dev/issue/XXXX): Remove after upgrading to containerd 1.5+
proto.RegisterType((*Options)(nil), "runtimeoptions.v1.Options")
```

Reply:
Actually the TODO comment should be more like:

```
// TODO(gvisor.dev/issue/XXXX): Upgrade runtimeoptions.proto after upgrading to containerd 1.5
```

Reply:
> I think a comment is necessary that says that this is a temporary fix to allow the shim to work with containerd 1.5+

Thanks! I wish my first contribution to gVisor felt like less of a hack 🙂 

> create a new Github issue to upgrade containerd dependency to 1.5 and add a TODO comment

Done!

> Please move this together with the other registration

Done! I kept it out of `runtimeoptions_cri.go` because I saw it was generated code.

Reply:
TBF, I figured adding this to the _cri.go file wouldn't be beneficial since this is a generated file and is copied from the containerd code. We would need to maintain the patch and might be likely to cause a regression when we update the file when we update to containerd 1.4.

Reply:
> might be likely to cause a regression when we update the file when we update to containerd 1.4.

Yeah, that's why I kept changes out of `_cri.go` initially. Updated the PR!